harness/determined/cli/user.py

```diff
@@ -313,6 +443,46 @@ def edit(args: argparse.Namespace) -> None:
                 help="grant/remove user admin permissions",
             ),
         ]),
+        cli.Cmd("token", None, "manage access tokens", [
```

Comment:
I don't think this belongs as a whole subtree of `det user`.  Nested subcommands makes the CLI harder to learn for end users.

Can we do something more like:
  - `det token login` (instead of `det user login --token`)
  - `det token describe`
  - `det token list`
  - `det token revoke`
  - `det token create`
  - `det token update`

Reply:
There was a conversation with @azhou-determined  [here](https://hpe-aiatscale.atlassian.net/browse/DET-10455?focusedCommentId=47492). Let us know your thoughts

Reply:
Resolving it as it is now implemented 
1. Moved from `det user token *` to `det token *` cli pattern 
2. We do not include SDK designs anymore in the PR
3. We now support Many:1 :: token:user relationship